xml/System.Web/HttpServerUtility.xml

```diff
@@ -701,6 +701,7 @@
  ]]></format>
         </remarks>
         <exception cref="T:System.Web.HttpException">The current <see cref="T:System.Web.HttpContext" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.Web.HttpException"><paramref name="path" /> is a physical path, but a virtual path was expected.</exception>
```

Comment:
The convention used throughout the managed reference is to have a single \<exception> element for a particular exception type and to list multiple exception conditions separated by "-or-". So there should be a single exception element for HttpException that appears as follows:

\<exception cref="T:System.Web.HttpException">The current \<see cref="T:System.Web.HttpContext" /> is \<see langword="null" />.

-or-

\<paramref name="path" /> is a physical path, but a virtual path was expected.\</exception> 

Would you like to make the change, or would you prefer that we make it for you?